sdk/template/Azure.Template/tests/Samples/Sample1_HelloWorld.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+#region Snippet:Azure_Template_Namespaces
+using Azure.Template.Models;
+#endregion
+using NUnit.Framework;
+
+namespace Azure.Template.Tests.Samples
+{
+    public partial class Sample1_HelloWorld: SamplesBase<TemplateClientTestEnvironment>
+    {
+        [Test]
+        [SyncOnly]
+        public void GettingASecret()
+        {
+            var endpoint = TestEnvironment.KeyVaultUri;
+
+            #region Snippet:Azure_Template_GetSecret
+#if SNIPPET
+            string endpoint = "https://myvault.vault.azure.net";
+#endif
+            var client = new TemplateClient(endpoint, new DefaultAzureCredential());
```

Comment:
This won't compile:

1. `endpoint1 is not a `Uri`. See my sample below for how to fix that.
2. This was a grow-up example and this should probably be a `SecretClient`.

I highly recommend using a real PR as opposed to a draft so it goes through all the normal checks which would've caught this. Draft PRs are more for when you're prototyping some idea and don't want it merged.

Reply:
Current codegen only generate according to the swagger definition, that is the endpoint is string. It is compiled successfully.